bot/code_review_bot/revisions.py

```diff
@@ -269,14 +269,36 @@ def analyze_patch(self):
         assert isinstance(self.patch, str), "Invalid patch type"
 
         # List all modified lines from current revision changes
-        patch = Patch.parse_patch(self.patch, skip_comments=False)
-        assert patch != {}, "Empty patch"
+        patch_stats = rs_parsepatch.get_diffs(self.patch)
+        assert patch_stats, "Empty patch"
+
+        def _get_lines(file_stats):
+            if file_stats["new"]:
+                # file addition
+                return [line[1] for line in file_stats["lines"]]
+            else:
+                # file modification
+                return sorted(
+                    set(
+                        [
+                            line[0] or line[1]
```

Comment:
Was the behavior with parsepatch the same? Was it returning the line number in the `before` state?

Reply:
Filed an issue on parsepatch: https://github.com/mozilla/pyo3-parsepatch/issues/18

I'm not sure who's to blame here

Reply:
I was able to use directly `get_lines` for most cases, and fall back to `get_diffs` for additions only.